Complex.c

```diff
@@ -0,0 +1,43 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Ayisha Sameera,GET.
+// ------------------------------------------------------------------
+// ClassicLibrary.c
+// Program on branch Test1.2
+// Library to perform arithmetic operations on complex numbers.
+// ------------------------------------------------------------------------------------------------
+
+#include <stdbool.h>
+#include "Complex.h"
+#include "math.h"
+
+ComplexNumber Op_Add (ComplexNumber num1, ComplexNumber num2) {
+   ComplexNumber result = { num1.Real + num2.Real , num1.Img + num2.Img };
+   return result;
+}
+
+ComplexNumber Op_Sub (ComplexNumber num1, ComplexNumber num2) {
+   ComplexNumber result = { num1.Real - num2.Real , num1.Img - num2.Img };
+   return result;
+}
+
+ComplexNumber Op_Mul (ComplexNumber num1, ComplexNumber num2) {
+   ComplexNumber result = { (num1.Real * num2.Real) - (num1.Img * num2.Img),
+    (num1.Img * num2.Real) + (num2.Img * num1.Real) };
+   return result;
+}
+
+ComplexNumber Conjugate (ComplexNumber num) {
+   num.Img = -num.Img;
+   return num;
+}
+
+double Modulus (ComplexNumber num) {
+   return sqrt (num.Real * num.Real + num.Img * num.Img);
+}
+
+bool IsEqual (ComplexNumber num1, ComplexNumber num2) {
+   // Check if difference between the two floating point numbers is negligibly small
+   return ((num1.Real - num2.Real < 0.0001) && (num1.Img - num2.Img < 0.0001));

```

Comment:
Use a const for 0.0001, if precision changes at some point, you won't have to manually ensure 0.0001 has been exchanged with the correct value